Moli/Core/Screens/Profile/ProfileImageAndBannerView.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+import Mantis
+
+enum ImageType {
+    case profile, banner
+}
+
+struct ProfileImageAndBannerView: View {
+    @EnvironmentObject private var myUser: MyUserDataModel
+    @StateObject var profileImagesDataModel = ProfileImagesDataModel()
+
+    var bannerImages: API.User.BannerThumbnailContainer?
+    var profileUserImage: API.User.UserMinified
+    var isMe: Bool
+    private let profileImageSizeYOffset: CGFloat = .userImageSizeMax / 2
+
+    private func onImageCropped (_ image: UIImage) {
+        Task {
+            await profileImagesDataModel.onImageCroppedSelected(image)
+            await myUser.user.refreshProfile()
+        }
+    }
+
+    private func onRemoveImage () {
+        Task {
+            await profileImagesDataModel.removeImage()
+            await myUser.user.refreshProfile()
+        }
+    }
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            HStack(spacing: 0) {
+                BannerImageView(bannerImages: bannerImages, isMe: isMe)
+            }
+            .frame(height: .bannerImageHeight)
+            .frame(maxWidth: .infinity)
+            .background(Color.gray50)
+            .onTapGesture {
+                if isMe {
+                    profileImagesDataModel.onImageTap(.banner)
+                }
+            }
+
+            VStack(spacing: 0) {
+                UserThumbnailView(userMinified: profileUserImage, size: .max)
+            }
```

Comment:
:art: Den här VStacken kan man ta bort, legacy från min sida
```suggestion
                UserThumbnailView(userMinified: profileUserImage, size: .max)
```